tests/unit/python/fledge/common/test_configuration_manager.py

```diff
@@ -2842,3 +2828,25 @@ def test_good__validate_value_per_optional_attribute(self, new_value_entry, stor
         except Exception:
             raised = True
         assert raised is False
+
+    def test__ignore_unsupported_key_in_config_items(self):
+        storage_client_mock = MagicMock(spec=StorageClientAsync)
+        c_mgr = ConfigurationManager(storage_client_mock)
+        raised_err = False
+        entry_name = "test"
+        item_name = "ignore_entry_name"
+        test_config = {
+            item_name: {
+                "description": "Test with entry_name",
+                "type": "string",
+                "default": "test_default_value",
+                entry_name: "some_value"
+
+            }
+        }
+        try:
+            c_mgr._validate_category_val(category_name=CAT_NAME, category_val=test_config,
+                                         set_value_val_from_default_val=True)
+        except Exception:
+            raised_err = True
+        assert raised_err is False
```

Comment:
`warning` will appear here ? 
We should suppress warning patch log and assert them